### PR TITLE
Cleanup and fixes for annotations

### PR DESCRIPTION
**Pull request by jdesrosiers**
This is mostly formatting changes, but also trims some things I didn't think were relevant. The main thing I wanted to change here was the suggestion that `default` could affect validation.

### CODE REVIEW

File: source/reference/generic.rst [hunk 1]
```diff
@@ -18,37 +18,40 @@ for all JSON types.
...
+Non-validation tools may use this information to provide a default
+value for a missing key/value pair, however validators should ignore
+the ``default`` keyword. The value of ``default`` should validate
+against the schema in which it resides, but that isn't required.
+
+|draft6| The ``examples`` keyword is a place to provide an array of
+examples that validate against the schema. This isn't used for
+validation, but may help with explaining the effect and purpose of the
+schema to a reader. Each entry should validate against the schema in
+which is resides, but that isn't strictly required. There is no need
```
Comment (karenetheridge) [inline source/reference/generic.rst:44]: not your error, but:

```suggestion
which it resides, but that isn't strictly required. There is no need
```

**Review by karenetheridge — APPROVED** [review]
This was a bit harder to review because of all the whitespace changes included. Would it be possible to do that in a separate commit?

**Comment by jdesrosiers** [comment]
You're right, I'll try to do whitespace changes in separate commits in the future. I'm also hoping to have build previews working well enough that reviewers can use that as well.

**Comment by jdesrosiers** [comment]
I also added documentation for the `readOnly` and `writeOnly` keywords that were added in draft-07, but not documented yet.

**Pull request merged**